<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameters</class>
 <widget class="QDialog" name="Parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelType">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kgon Type</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxType">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Regular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dependent</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="Line" name="lineV">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelSizeBbox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Polygon Bbox</string>
     </property>
     <property name="buddy">
      <cstring>lineEditBbox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineEditBbox">
     <property name="text">
      <string>1/1</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSize">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kgon Size</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxSize</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxSize">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelSizeP">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Polygon Size</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxSizeP</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinBoxSizeP">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="lineH">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelRadius">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRadius</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditRadius">
     <property name="text">
      <string>1/1</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelRadiusRatio">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>= BBox ratio</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRadiusRatio</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="lineEditRadiusRatio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1/1</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QCheckBox" name="checkBoxRadiusRatio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Keep ratio</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelEpsilon">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Epsilon</string>
     </property>
     <property name="buddy">
      <cstring>lineEditEpsilon</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditEpsilon">
     <property name="text">
      <string>1/10</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="labelEpsilonRatio">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>= Radius ratio</string>
     </property>
     <property name="buddy">
      <cstring>lineEditEpsilonRatio</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEditEpsilonRatio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1/10</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QCheckBox" name="checkBoxEpsilonRatio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Keep ratio</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDelta">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delta</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDelta</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditDelta">
     <property name="text">
      <string>1/40</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="labelDeltaRatio">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>= Epsilon ratio</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDeltaRatio</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEditDeltaRatio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1/4</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QCheckBox" name="checkBoxDeltaRatio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Keep ratio</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditRadius</tabstop>
  <tabstop>lineEditRadiusRatio</tabstop>
  <tabstop>lineEditEpsilon</tabstop>
  <tabstop>lineEditEpsilonRatio</tabstop>
  <tabstop>lineEditDelta</tabstop>
  <tabstop>lineEditDeltaRatio</tabstop>
  <tabstop>checkBoxRadiusRatio</tabstop>
  <tabstop>checkBoxEpsilonRatio</tabstop>
  <tabstop>checkBoxDeltaRatio</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>spinBoxSize</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>lineEditBbox</tabstop>
  <tabstop>spinBoxSizeP</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Parameters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Parameters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Parameters</receiver>
   <slot>editType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Parameters</receiver>
   <slot>editSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRadius</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEpsilon</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editEpsilon(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDelta</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editDelta(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRadius</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedRadius()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEpsilon</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedEpsilon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDelta</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRadiusRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>Parameters</receiver>
   <slot>keepRadiusRatio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEpsilonRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>Parameters</receiver>
   <slot>keepEpsilonRatio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDeltaRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>Parameters</receiver>
   <slot>keepDeltaRatio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRadiusRatio</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editRadiusRatio(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEpsilonRatio</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editEpsilonRatio(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDeltaRatio</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Parameters</receiver>
   <slot>editDeltaRatio(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRadiusRatio</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedRadiusRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEpsilonRatio</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedEpsilonRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDeltaRatio</sender>
   <signal>editingFinished()</signal>
   <receiver>Parameters</receiver>
   <slot>changedDeltaRatio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editType(int)</slot>
  <slot>editSize(int)</slot>
  <slot>editRadius(QString)</slot>
  <slot>editEpsilon(QString)</slot>
  <slot>editDelta(QString)</slot>
  <slot>changedRadius()</slot>
  <slot>changedEpsilon()</slot>
  <slot>changedDelta()</slot>
  <slot>keepRadiusRatio(bool)</slot>
  <slot>keepEpsilonRatio(bool)</slot>
  <slot>keepDeltaRatio(bool)</slot>
  <slot>editRadiusRatio(QString)</slot>
  <slot>editEpsilonRatio(QString)</slot>
  <slot>editDeltaRatio(QString)</slot>
  <slot>changedRadiusRatio()</slot>
  <slot>changedEpsilonRatio()</slot>
  <slot>changedDeltaRatio()</slot>
 </slots>
</ui>
